### PR TITLE
[Cloudflare] Fix recent CF JS Challenge error that calls DOM

### PR DESCRIPTION
**Pull request by DitFranXX**
From Anorov/cloudflare-scrape#206
Related issues: inorichi/tachiyomi-extensions#951

Recent CF changes research: https://github.com/inorichi/tachiyomi-extensions/issues/951#issuecomment-475467259

Ps. [`atob` package in NPM reaches ~780K downloads/week](https://www.npmjs.com/package/atob)

### CODE REVIEW

**Comment by DitFranXX** [comment]
Seems they reverted it or not enabled on my side.. I didn't get any `div` now

File: app/src/main/java/eu/kanade/tachiyomi/network/CloudflareInterceptor.kt [hunk 1]
```diff
@@ -80,4 +90,42 @@ class CloudflareInterceptor : Interceptor {
         }
     }
 
+    companion object {
+        // atob() is browser API, Got this from NPM package, `atob`.
```
Comment (arkon) [inline app/src/main/java/eu/kanade/tachiyomi/network/CloudflareInterceptor.kt]: Doesn't Kotlin or Java have their own base64 encoding/decoding functions?

Comment (Peuuuur-Noel) [reply app/src/main/java/eu/kanade/tachiyomi/network/CloudflareInterceptor.kt]: You can maybe use Kotlin base64 decode function in Javascript with https://github.com/square/duktape-android#calling-java-from-javascript

Comment (DitFranXX) [reply app/src/main/java/eu/kanade/tachiyomi/network/CloudflareInterceptor.kt]: Updated on `eba3e52`

Comment (inorichi) [reply app/src/main/java/eu/kanade/tachiyomi/network/CloudflareInterceptor.kt]: I think Okio has a base64 decode function. Please use that if possible. I prefer that to android dependencies

Comment (DitFranXX) [reply app/src/main/java/eu/kanade/tachiyomi/network/CloudflareInterceptor.kt]: Updated on de8e950

**Pull request merged**